core/src/main/resources/reference.conf

```diff
@@ -314,6 +325,17 @@ services {
       # db-flush-rate = 5 seconds
     }
   }
+  HealthMonitor {
+    class = "cromwell.services.healthmonitor.impl.standard.StandardHealthMonitorServiceActor"
+    # Override the standard dispatcher. In particular this one has a low throughput so as to be lower in priority
+    dispatcher = "akka.dispatchers.health-monitor-dispatcher"
+    # How long to wait between status check sweeps
```

Comment:
Saw @kshakir mention in another PR that this belongs in the example conf, which not only do I like but this solves a problem I had anyways

Reply:
done